build/ci/phase-template.yml

```diff
@@ -35,8 +37,17 @@ phases:
     - ${{ if eq(parameters.name, 'MacOS') }}:
       - script: brew update && brew install libomp mono-libgdiplus gettext && brew link gettext --force
         displayName: Install runtime dependencies
+    - script: $(_buildScript) -$(_configuration) -runtests -coverage=true
```

Comment:
Why can't this be a single script?

`- script: $(_buildScript) -$(_configuration) -runtests -coverage=$(_codeCoverage)`

Reply:
Good idea.